docs/topics/collections-overview.md

```diff
@@ -46,7 +46,7 @@ Although, if you try to reassign a `val` collection, you'll get a compilation er
 fun main() {
 //sampleStart
     val numbers = mutableListOf("one", "two", "three", "four")
-    numbers.add("five")   // this is OK    
+    println(numbers.add("five"))   // this is OK    
```

Comment:
This prints `true`, not a really demonstrative output.
If you want to show the change, print the whole collection in a separate call of println

Reply:
I added this printing for the purpose that if launching this example, then there's some result, but not only the progress and then nothing. I suppose we should either make such examples with no output not launchable or, if to leave launchable, to print something.
We can print the whole collection, I'll ad this.